src/js/components/Drop/stories/Custom.js

```diff
@@ -7,6 +7,8 @@ import { deepMerge } from 'grommet/utils';
 const customTheme = deepMerge(grommet, {
   global: {
     drop: {
+      // margin: '10px',
```

Comment:
Remove these commented out lines if we aren't using them

Reply:
Yeah I left them there for testing if someone pulled down the branch but should be taken out when its merged